cmd/auth/profiles.go

```diff
@@ -45,13 +44,7 @@ func (c *profileMetadata) Load(ctx context.Context, skipValidate bool) {
 	}
 
 	if skipValidate {
-		err := cfg.Authenticate(&http.Request{
-			Header: make(http.Header),
-		})
-		if err != nil {
-			return
-		}
-		c.Host = cfg.Host
+		c.Host = cfg.CanonicalHostName()
 		c.AuthType = cfg.AuthType
```

Comment:
The `AuthType` will only be set if explicitly set (environment of in `.databrickscfg` profile).

It is guaranteed to be set only if we run `Authenticate` to pull a token.